supp2u/src/components/Business/BusinessSingleView.js

```diff
@@ -30,23 +25,23 @@ function BusinessSingleView(props) {
             setInfo(res.data)
         })
         .catch(err =>{
-            console.log('ERROR POST\n', err)
+            console.log('ERROR GETTING BUSINESS\n', err)
         })
         //api GET to bring in the rating
         Axios.get(`${process.env.REACT_APP_BACKEND_URL}businesses/${businy}/ratings`)
         .then(res =>{
             setRating(res)
         })
         .catch(err =>{
-            console.log('ERROR POST\n', err)
+            console.log('ERROR GETTING RATINGS\n', err)
```

Comment:
Looks good, I like that `\n` at the end of your label to make parsing error easier 👍